cookbook/Cookbook-file-upload-saving-files-to-server.md

```diff
@@ -48,7 +56,7 @@ mkYesod "App" [parseRoutes|
 |]
 
 instance Yesod App where
-    maximumContentLength _ (Just ImagesR) = Just $ 200 * 1024 * 1024 -- 200 megabytes
+    maximumContentLength _ _ = Just $ 200 * 1024 * 1024 -- 200 megabytes
```

Comment:
@psibi Actually, maybe I should have changed it to be something more explicit? :

```haskell
maximumContentLength _ (Just ImagesR) = Just $ 200 * 1024 * 1024 -- 200 megabytes
maximumContentLength _ _ = Just $ 10 * 1024 * 1024 -- 10 megabytes
```